lib/spark.js

```diff
@@ -83,20 +83,27 @@ export default class Spark {
     }
   }
 
-  async fetchCAR (protocol, address, cid, stats) {
+  async fetchCAR (protocol, address, cid, stats, { maxFetchDurationMs = MAX_REQUEST_DURATION_MS } = {}) {
```

Comment:
inconsistent naming, should be either max fetch duration or max request duration everywhere